ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/KafkaStreamsStateCount.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.entity;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonValue;
+import io.confluent.ksql.util.KsqlConstants;
+import io.confluent.ksql.util.KsqlException;
+
+import java.util.Comparator;
+import java.util.Map;
+import java.util.Objects;
+import java.util.TreeMap;
+
+import org.apache.kafka.streams.KafkaStreams;
+
+/**
+ * Used to keep track of a the state of KafkaStreams application
+ * across multiple servers. Used in {@link RunningQuery}.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class KafkaStreamsStateCount {
+  
+  // Use a TreeMap so toString() will always return the same string
+  private final TreeMap<KafkaStreams.State, Integer> state;
```

Comment:
I believe you can use `EnumMap` here - as the order is deterministic based on the ordering of the State enum ordinal